qemu/tests/cfg/timerdevice.cfg

```diff
@@ -41,6 +41,13 @@
                 - base_localtime:
                     only Windows
                     rtc_base = localtime
+                - base_date:
+                    Windows:
+                        only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
+                    check_chronyd_active_cmd = "service chronyd status | grep 'active (running)'"
+                    mask_chronyd_cmd = "systemctl stop chronyd && systemctl mask chronyd"
+                    unmask_chronyd_cmd = "systemctl unmask chronyd"
```

Comment:
Line 47-49 is for `linux` only, please add `Linux:` for these params, then you don't have to compare `os_type` in `timerdevice_boot.py`.